src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/presubmit/UpdateHearingRequestSubmit.java

```diff
@@ -88,6 +88,7 @@ public PreSubmitCallbackResponse<AsylumCase> handle(
             try {
                 hearingService.updateHearing(
                     updateHearingPayloadService.createUpdateHearingPayload(
+                        asylumCase,
```

Comment:
why do we need to pass asylumCase object instead of just hearingFacilities? if you think there is a benefit in passing the object, then can we refactor this method to remove hearingChannels and Locations as arguments, instead it could be read inside the method from asylumcase object

Reply:
I already did the refactor, please have a look at it.